.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - stable
+  - 8.9.4
+
+
+
+cache:
+directories:
+  - "node_modules"
+
+
+before_script:
+
+  - psql -c 'create database travis;' -U postgres
+
+install:
+  - npm install
```

Comment:
There is no need for line 17-18. Since you have specified the language and node version, Travis will automatically run npm install after installing node and npm.

Reply:
I will remove this then